source/common/util/generate-file-link.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * BEGIN HEADER
+ *
+ * Contains:        Utility function
+ * CVM-Role:        <none>
+ * Maintainer:      Yhancik
+ * License:         GNU GPL v3
+ *
+ * Description:     Generates Markdown file link or image
+ *
+ * END HEADER
+ */
+
+const path = require('path')
```

Comment:
Since we had to disable node integration, this won't work in the renderer process. Instead, what we can do now is use `const path = window.path`. (And that won't work in the main process, so since this function is only called from within the MarkdownEditor instance, I'd rather put this function there. Then we can also upgrade it to ES6 using `export default function` and using import statements)

Reply:
Update: Just saw that the function is being required in both main and renderer process; so I think we'll need the following (ugly) construct:

`const path = (typeof window !== 'undefined') ? window.path : require('path')`